### PR TITLE
feat: 2FA is now available in all plans

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: Title
Ticket: MEN-5677

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

File: 02.Overview/11.Security/docs.md [hunk 1]
```diff
@@ -69,11 +69,10 @@ You can find more information in the
 
 As a user, you interact with your Devices either via API calls issued to the
 Mender server, or via the web UI. In both cases, the connection is HTTPS
```
Comment (merlin-northern) [inline 02.Overview/11.Security/docs.md:71]: suggested change: "Mender server" -> "Mender _Server_" I think we address it with capital letters everywhere.

File: 02.Overview/11.Security/docs.md [hunk 2]
```diff
@@ -69,11 +69,10 @@ You can find more information in the
...
-encrypted with TLS. To authenticate with the Mender server, a user must log in
-by presenting a valid email address and password, as well as a Two Factor
-authentication code (if available in the plan and enabled). The client
-(e.g. web UI) used to log in then receives a JWT token with a default expiration
-time of one week.
+encrypted with TLS. A user must log in to authenticate with the Mender server
+by presenting a valid email address, password, and a Two Factor authentication
```
Comment (merlin-northern) [inline 02.Overview/11.Security/docs.md:73]: suggested change: "Two Factor authentication" -> "_two-factor_ authentication" no reason for the capital letters here?

File: 02.Overview/11.Security/docs.md [hunk 3]
```diff
@@ -69,11 +69,10 @@ You can find more information in the
...
+code if enabled. The client (e.g., web UI) used to log in then receives a JWT token
```
Comment (merlin-northern) [inline 02.Overview/11.Security/docs.md:74]: suggested change: "code" -> "_token_"

Comment (merlin-northern) [reply 02.Overview/11.Security/docs.md:74]: this sentence "The client used to log in then receives a JWT token with a ..." does not make sense to me.

**Review by merlin-northern — CHANGES_REQUESTED** [review]

**Review by kjaskiewiczz — APPROVED** [review]

**Review by 0lmi — APPROVED** [review]
LGTM

**Comment by tranchitella** [comment]
@merlin-northern ping

**Comment by merlin-northern** [comment]
> @merlin-northern ping

PONG
yes I am here. sorry for the delay. on it.

File: 02.Overview/11.Security/docs.md [hunk 4]
```diff
@@ -68,12 +68,11 @@ You can find more information in the
...
 
 As a user, you interact with your Devices either via API calls issued to the
-Mender server, or via the web UI. In both cases, the connection is HTTPS
-encrypted with TLS. To authenticate with the Mender server, a user must log in
-by presenting a valid email address and password, as well as a Two Factor
-authentication code (if available in the plan and enabled). The client
-(e.g. web UI) used to log in then receives a JWT token with a default expiration
-time of one week.
+Mender Server, or via the web UI. In both cases, the connection is HTTPS
+encrypted with TLS. A user must log in to authenticate with the Mender server
```
Comment (merlin-northern) [inline 02.Overview/11.Security/docs.md:72]: let's try to be consistent with the 'Mender Server' and 'Mender server' at least in those two lines.

**Review by merlin-northern — APPROVED** [review]
pre-approving not to block, one byte change requested. sorry for being picky.

**Review by merlin-northern — APPROVED** [review]

**Pull request merged**